gnovm/pkg/gnolang/precompile_test.go

```diff
@@ -1,94 +1,286 @@
 package gnolang
 
 import (
-	"bytes"
-	"errors"
-	"go/format"
-	"go/parser"
-	"go/token"
+	"go/ast"
+	"strings"
 	"testing"
 
+	"github.com/jaekwon/testify/require"
 	"github.com/stretchr/testify/assert"
```

Comment:
Since we agreed on using `jaekwon/testify` https://github.com/gnolang/gno/issues/777#issuecomment-1632014195

Maybe this PR is the right place to change it to `jaekwon/testify` as well?

```suggestion
	"github.com/jaekwon/testify/assert"
```

Reply:
Good catch, thanks @harry-hov, note that this also requires switch in the expected/got parameters, see [54b9a47](https://github.com/gnolang/gno/pull/1659/commits/54b9a4722a8e22de73d2a606e0fefeb551c1adbf).